### PR TITLE
Chore/update rise storage

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees Updated to use the latest version of the Storage component. Also added integration tests for the Widget and eliminated the e2e tests. Please review. Thx.

File: test/integration/widget-image.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      test("should set background image", function() {
+        assert.equal(document.getElementById("image").style.backgroundImage,
+          'url(https://www.googleapis.com/storage/v1/b/risemedialibrary-b428b4e8-c8b9-41d5-8a10-b4193c789443/o/Widgets%2Fmoon.jpg?alt=media)');
+      });
+
+      test("should refresh background image", function() {
+        clock.tick(300000);
+
+        assert(spy.calledTwice);
+        console.log(document.getElementById("image").style.backgroundImage);
```
Comment (stulees) [inline test/integration/widget-image.html]: @donnapep Did you want to leave this log statement in? Also, the check that `generateRequest()` gets called twice seems like a good consolation, assuming you had no luck in verifying the backgroundImage value, correct?

File: src/widget/storage.js [hunk 2]
```diff
@@ -14,19 +15,28 @@ RiseVision.Image.Storage = function (params) {
...
+          img.style.backgroundImage = "url(" + e.detail.url + ")";
+        }
+
         RiseVision.Image.ready();
         isLoading = false;
       }
+      else {
+        // Image has been changed.
+        if (e.detail.hasOwnProperty("changed") && e.detail.changed) {
+          if (e.detail && e.detail.url) {
```
Comment (stulees) [inline src/widget/storage.js]: @donnapep Shouldn't `if(e.detail && e.detail.url)` be the initial check before the outer `e.detail.changed`?

File: gulpfile.js [hunk 3]
```diff
@@ -32,7 +33,7 @@
   });
 
   gulp.task("config", function() {
-    var env = process.env.NODE_ENV || "prod";
+    var env = process.env.NODE_ENV || "dev";
```
Comment (stulees) [inline gulpfile.js]: @donnapep I think you meant to switch this back to `"prod"`

File: dist/widget.html [hunk 4]
```diff
@@ -62,7 +62,7 @@
 </head>
 
 <body>
-  <rise-storage refresh="5"></rise-storage>
+  <rise-storage refresh="5" env="test"></rise-storage>
```
Comment (stulees) [inline dist/widget.html]: @donnapep I don't see the `src/widget.html` changed, so maybe you need a rebuild? Doesn't seem like you should be setting `env` directly on the component as its handled in your logic.

**Comment by donnapep** [comment]
@stulees I think I fixed everything you mentioned. Can you review again? Thx.

**Comment by stulees** [comment]
@donnapep :+1: (assuming a rebase), thanks

**Pull request merged**